src/nyc_trees/apps/home/views.py

```diff
@@ -21,6 +24,28 @@ def retrieve_job_status(request):
     pass
 
 
+def redirect_to_flat_page(url):
```

Comment:
The function that this calls is how flatpage url/view routing works under the hood.